libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -431,6 +431,16 @@ class AC_AttitudeControl {
     // get the value of the PD scale that was used in the last loop, for logging
     const Vector3f &get_PD_scale_logging(void) const { return _pd_scale_used; }
```

Comment:
Like wise this can be removed or private now.

Reply:
removed